File: Day_24/Q94.java
import java.util.*;

class Q94{
    static Scanner sc = new Scanner(System.in);

    //Function To Compress String
    static String compress_string(String str){
        String ans = "";

        for(int i = 0;i < str.length();i++){

            int count = 1;

            //Count Same Characters
            while(i < str.length() - 1 && str.charAt(i) == str.charAt(i + 1)){
                count++;
                i++;
            }

            ans += str.charAt(i);
            ans += count;
        }

        return ans;
    }

    public static void main(String args[]){
        String str = sc.nextLine();

        //Print Result
        System.out.println(compress_string(str));

        sc.close();
    }
}

File: Day_24/Q94.cpp
#include<bits/stdc++.h>
using namespace std;

//Function To Compress String
string compress_string(string str){
    string ans = "";

    for(int i = 0;i < str.length();i++){

        int count = 1;

        //Count Same Characters
        while(i < str.length() - 1 && str[i] == str[i + 1]){
            count++;
            i++;
        }

        ans += str[i];
        ans += to_string(count);
    }

    return ans;
}

int main(){
    string str;
    getline(cin,str);

    //Print Result
    cout << compress_string(str);

    return 0;
}